plextraktsync/trakt_api.py

```diff
@@ -334,15 +336,16 @@ def queue_size(self):
 
         return size
 
-    def flush(self):
+    def flush(self, force=False):
         """
-        Flush the queue if it's bigger than batch_size
+        Flush the queue every batch_delay seconds
         """
-        if not self.batch_size:
+        if not self.batch_delay and force is False:
             return
-
-        if self.queue_size() >= self.batch_size:
+        elapsed = time() - self.last_sent_time
+        if elapsed >= self.batch_delay or force is True:
             self.submit_collection()
+            self.last_sent_time = time()
```

Comment:
move `self.last_sent_time` updating to `self.submit_collection()`?

Reply:
thanks, it's done